backend/src/investigation/mod.rs

```diff
@@ -2,7 +2,7 @@ mod api;
 pub(super) mod repository;
 pub(super) mod structs;
 
-pub(crate) use self::{
+pub use self::{
     api::router, repository::list_investigations_for_committee_session,
     structs::PollingStationInvestigation,
 };
```

Comment:
We [want the integration tests to use json](https://github.com/kiesraad/abacus/blob/main/documentatie/ontwikkelproces/test-tooling.md#integratietests), which is why this struct was not made visible outside of the crate